### PR TITLE
Add draft features for HTML elements

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
This would really benefit from https://github.com/web-platform-dx/web-features/issues/1018, so that we don't need to carefully review exactly when each of these were added.

File: features/area.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+draft: true
+name: <area>
```
Comment (foolip) [inline features/area.yml:2]: This features should be Image maps with https://html.spec.whatwg.org/multipage/image-maps.html as the spec link.

File: features/draft/html-elements/video.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+draft_date: 2024-06-05
```
Comment (ddbeck) [inline features/draft/html-elements/video.yml:1]: Hmm, why doesn't this one have a dist?

**Comment by foolip** [comment]
@ddbeck this is ready for review now. I could keep tinkering with it indefinitely, but it's good enough now.

**Review by ddbeck — APPROVED** [review]

**Pull request merged**